### ISSUE TITLE
delete finished/ failed jobs from database??

### ISSUE THREAD

**Issue by jorgeakanieves** [spark-jobserver/spark-jobserver#585]
is any possibility to remove finished/ failed jobs from database? In case that executing several times the same job (same appName) and quering the api rest, it returns the same appName (className) but with different status (FINISHED/ RUNNING/ FAILED) so it´s a bit complicated to check the "current" status of the appName...

Any way to do this via API? (Not a workaround to perfom this like bash script, jdbc client to h2,...)

**Comment by noorul** [comment]
Are you not using jobId which is unique?

**Comment by jorgeakanieves** [comment]
Over the same appName (same jar)  I execute several iterations so always a new jobid is created.. And the jobs list becomes huge and can't get the last execution unless I capture and control the time of execution and this is not a very smart solution...

**Comment by noorul** [comment]
@jorgeakanieves Which version of SJS are you using?

**Comment by jorgeakanieves** [comment]
I´m deploying jobserver with velvia/spark-jobserver:0.6.2.mesos-0.28.1.spark-1.6.1 docker. 

Another issue is that sometimes, running jobserver against yarn by yarn-client mode, jobs are finishing in yarn (by an error or other fact) but it doesn´t finish in jobserver (there´s a sync problem) so I can´t stop it in jobserver and always stay in running status.

¿?

**Comment by BigFishBall** [comment]
Hi ，
     Is the problem resolved? I have the same issue: is there any way to clean up job list via APIs？
     This job list become Huge, and ... I search no answer.

thanks.

**Comment by BigFishBall** [comment]
I deleted /tmp/spark-jobserver before starting the jobserver.
and joblist is cleanup。
Refer to https://github.com/spark-jobserver/spark-jobserver/issues/516#issuecomment-227986726

But I still hope there is a easy way~~   :p

**Comment by Nibooor** [comment]
See https://github.com/spark-jobserver/spark-jobserver/pull/1375
